Client/Application/AppDelegate.swift

```diff
@@ -189,10 +190,88 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         // Run an invalidate when we come back into the app.
         profile.panelDataObservers.activityStream.invalidate(highlights: true)
 
+        setUpDeepLinks(application: application)
+
         log.debug("Done with setting up the application.")
         return true
     }
 
+    func setUpDeepLinks(application: UIApplication) {
```

Comment:
Agree we should refactor this into somewhere else but not sure LeanplumIntegration is a great place. If you don't find it another home, once again create a bug and include a reference to it in a comment.